config/config.ini.example

```diff
@@ -1,27 +1,76 @@
 # Uncomment a line (remove the #) when you want to change its default value.
-# Multiple arguments can be listed as [arg1, arg2, ... ]
-# Number of arguments must match manager_count or be a single arguement (single arguements will apply to all Managers)
-# To exclude an argument for a specific manager, use 'None'
-
-# Server Settings
-#debug											# Enables debugging mode
-#host:											# Address to listen on (default 127.0.0.1)
-#port:											# Port to listen on (default: 4000)
-#concurrency: 200                                # Maximum concurrent connections for the webserver(default: 200)
-#manager_count: 1								# Number of Managers to run. (default: 1)
-
-# Manager-Specific Settings
-#manager_name: manager_0                        # Name of the Manager in the logs. Default(manager_0).
-#key:                                           # Google Maps API Key to use
-#filters:										# File containing filter settings (default: filters.json)
-#alarms: 										# File containing alarm settings (default: alarms.json)
-#rules: 										# File containing rules settings (default: None)
-#geofence:										# File containing geofence(s) used to filter (default: None)
-#location:										# Location for the manager. 'Name' or 'lat lng' (default: None)
-#locale:										# Language to be used to translate names (default: en)
-#cache_type:                                    # Specify the type of cache to use. Options: ['mem', 'file'] (Default: 'mem')
-#unit:											# Units used to measure distance. Either 'imperial' or 'metric' (default: imperial)
-#timelimit:										# Minimum number of seconds remaining to send a notification (default: 0)
-#max_attempts:									# Maximum number of attempts an alarm makes to send a notification. (default: 3)
-#timezone:                                      # Timezone used for notifications Ex: 'America/Los_Angeles' or '[America/Los_Angeles, America/New_York]'
\ No newline at end of file
+# By default, PA will use `config/config.ini` to load settings.
+# You can override the config file selction with `--config-file` or `-cf`.
+
+
+########################
+# Webserver Settings
+########################
+
+#host: 127.0.0.1                # Interface to listen on (default='127.0.0.1')
+#port: 4000						# Port to listen on (default='4000')
+#concurrency: 200               # Maximum concurrent connections to webserver (default=200)
+#manager_count: 1				# Number of Managers to run (default=1)
+#debug                          # Enable debug logging (default='False)
+
+
+#########################
+# Manager Settings
+#########################
+# All of the settings below this line are manager-specific.
+# If a single setting is supplied, it will apply to all Managers.
+# Example: `locale: en` will set all Managers to english
+# If an array of settings is supplied, they will be apply to Managers in order.
+# Example: `locale: [ en, fr, sp ]` sets a different langauge for 3 Managers.
```

Comment:
`sp` does not exist, I think you want to say spanish so the correct option would be `es`